### ISSUE TITLE
About the performence based on our dataset

### ISSUE THREAD

**Issue by gaowq2017** [Yang-Bob/PMMs#5]
Hi Yang-Bob,
    I train  PMM and RPMM model based on our dateset, but the miou is low about 0.1. One example(two defect) of our dataset as follow:
![image](https://user-images.githubusercontent.com/29697367/91940172-9f489e80-ed29-11ea-97fd-dec539954df8.png)
![image](https://user-images.githubusercontent.com/29697367/91940366-e9ca1b00-ed29-11ea-8406-e325c898ceff.png)

Do you any good ideas? Thanks

**Comment by Yang-Bob** [comment]
Hi!

I think 0.1 is within the range of allowable error for the data is randomly chosen by the data loader.

Is the dataset in which you use fracture segmentation? Maybe you can read some paper or code to find whether there are works to handle this problem which is a better way in your dataset. If you want to use the few-shot segmentation method in your task, you may config your datasets to the format of the few-shot dataset. Note that the training set may be similar to your test set, e.g. it is hard to use the feature of animals to segment the fracture.
